docs/rfcs/XeTile.md

```diff
@@ -185,12 +232,275 @@ For example, for the tile size [128, 128] and sg_data [32, 128], along the secon
+|tile_transpose	| operation ::=XeTile.tile_transpose attr_dict $permuation_dims $vec : type($vec) -> type($res)	 | %vector_a = XeTile.transpose_tile %vector_b  #mp_b #mp_a: vector<64x32xfloat> into vector<32x64xfloat>  |
+|tile_reduce	| operation ::=XeTile.tile_reduce $kind $src attr_dict $reduction_dims: type($value) -> type($res)	 | %vector_a = XeTile.tile_reduce <add> %vector_b [1] #mp_b #mp_a: vector<64x32xfloat> into vector<64x1xfloat>  |
+|tile_broadcast	| operation ::=XeTile.tile_broadcast $src attr_dict $broadcast_dims: type($value) -> type($res)	 | %vector_a = XeTile.tile_broadcast %vector_b #mp_b #mp_a [0]: vector<1x32xfloat> into vector<64x32xfloat>  |
+|tile_conv_layout	| operation ::=XeTile.conv_layout attr_dict $src : type($value) -> type($res)	 | %vector_a = XeTile.tile_conv_layout %vector_b #mp_b #mp_a : vector<256x256xfloat> into vector<256x256xfloat>  |
+
 
-Within the `xetile.sg_map`, `wi_layout` specifies the layout in which WI threads correspond to the memory, and `wi_data` describes the data block accessed by each WI thread. In the example above, wi_layout=[2, 8] means that each subgroup has 16 WI threads in 2 rows and 8 columns, and wi_data=[1,2] means that each WI thread owns a [1,2] data fragment. The data elements with each data fragment assigned to a WI thread must be contiguous. So the sg_map describes a total [2,16] submatrix at the subgroup level.
+With these attributes, `tile_mma` does a matrix multiplication at a work group level vector. 
+```mlir
+   #wg_map_a = #xetile.wg_map<sg_layout = [8, 4], sg_data = [32, 32]>
+   #wg_map_b = #xetile.wg_map<sg_layout = [8, 4], sg_data = [32, 64]>
+   #wg_map_c = #xetile.wg_map<sg_layout = [8, 4], sg_data = [32, 64]>
 
-The size of `sg_data` within `xetile.wg_map` must be divisible by sg_map size, which equals to `wi_layout` multiplying with `wi_data` within `xetile.sg_map`. More specifically, for each dimension, the `sg_data` size must be divisible by `wi_layout` x `wi_data`. The sg_data size must be larger than or equal to the sg_map size. When the 2D subtensor size is larger than the sg_map size, it is distributed to WI threads in a round-robin fashion.
+   %vector_c = XeTile.tile_mma #wg_map_c #wg_map_a #wg_map_b #wg_map_c %vector_c, %vector_a, %vector_b :
+     vector<256x256xfloat>, vector<256x256xbf16>, vector<256x256xbf16>
+	   into vector<256x256xfloat>  
+```
 
+`tile_reduce` follows the vector.multi-reduction semantics and can be applied to 4D vector. 
+```mlir
+   #wg_map_a = #xetile.wg_map<sg_layout = [32, 1], sg_data = [8, 128]>
+   #wg_map_a2 = #xetile.wg_map<sg_layout = [32, 1], sg_data = [8, 1]>
+   %vector_a = XeTile.tile_reduce #wg_map_a #wg_map_a2  <add> %vector_b [1]: vector<256x128xfloat> into vector<256x1xfloat>
```

Comment:
How about using the following format?
`%vector_a = XeTile.tile_reduce <add> %vector_b[1] {#wg_map_a, #wg_map_a2}: vector<256x128xfloat> into vector<256x1xfloat>`

Reply:
OK

Reply:
fixed